segmentation_models_pytorch/decoders/fpn/decoder.py

```diff
@@ -51,36 +53,37 @@ def forward(self, x):
 
 
 class MergeBlock(nn.Module):
-    def __init__(self, policy):
+    def __init__(self, policy: Literal["add", "cat"]):
         super().__init__()
         if policy not in ["add", "cat"]:
             raise ValueError(
                 "`merge_policy` must be one of: ['add', 'cat'], got {}".format(policy)
             )
         self.policy = policy
 
-    def forward(self, x):
+    def forward(self, x: List[torch.Tensor]) -> torch.Tensor:
```

Comment:
Technically `List` is a bit too strict, it could be any `collections.abc.Sequence`. This includes things like tuples. Likely true for a lot of other places in the code base as well.